### PR TITLE
Bump version to v1.0.0rc2

### PR DESCRIPTION
**Pull request by fangyixiao18**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,11 @@ The master branch works with **PyTorch 1.6** or higher.
 
 ## What's New
 
+MMSelfSup **v1.0.0rc2** was released in 11/10/2022.
+
+- Full support of `MAE`, `SimMIM`, `MoCoV3`.
```
Comment (tonysy) [inline README.md]:
```suggestion
- Full support of `MAE`, `SimMIM`, `MoCoV3` with different model sizes and different pre-training epochs.
```

**Review by tonysy — COMMENTED** [review]
As inline comments

File: README_zh-CN.md [hunk 2]
```diff
@@ -65,7 +65,12 @@ MMSelfSup 是一个基于 PyTorch 实现的开源自监督表征学习工具箱
 
 ## 更新
 
-最新的 **v1.0.0rc1** 版本已经在 2022.09.01 发布。
+**v1.0.0rc2** 版本已经在 2022.10.11 发布。
+
+- `MAE`, `SimMIM`, `MoCoV3` 算法模型的全量支持。
```
Comment (tonysy) [inline README_zh-CN.md]:
```suggestion
- `MAE`, `SimMIM`, `MoCoV3` 算法模型的全量支持（支持多种模型尺寸和预训练轮次）。
```

File: docs/en/notes/changelog.md [hunk 3]
```diff
@@ -2,6 +2,44 @@
...
+
+- Fix classification configs ([#488](https://github.com/open-mmlab/mmselfsup/pull/488))
+- Fix MAE config name error ([#498](https://github.com/open-mmlab/mmselfsup/pull/498))
+
+#### Improvements
+
+- Refactor colab tutorial ([#470](https://github.com/open-mmlab/mmselfsup/pull/470)))
+- Update readthedocs requirements ([#472](https://github.com/open-mmlab/mmselfsup/pull/472))
+- Update CI ([#476](https://github.com/open-mmlab/mmselfsup/pull/476))
+- Refine mim_slurm_test.sh ([#477](https://github.com/open-mmlab/mmselfsup/pull/477))
```
Comment (YuanLiuuuuuu) [inline docs/en/notes/changelog.md]: Not just this file, but also other related files. Please check this [PR](https://github.com/open-mmlab/mmselfsup/pull/477)

**Review by tonysy — APPROVED** [review]
LGTM

**Review by YuanLiuuuuuu — APPROVED** [review]

**Pull request merged**